tests/integration-tests/tests/scaling/test_scaling.py

```diff
@@ -69,31 +72,195 @@ def test_multiple_jobs_submission(scheduler, region, pcluster_config_reader, clu
 @pytest.mark.usefixtures("region", "os", "instance")
 @pytest.mark.nodewatcher
 def test_nodewatcher_terminates_failing_node(scheduler, region, pcluster_config_reader, clusters_factory, test_datadir):
-    cluster_config = pcluster_config_reader()
+    initial_queue_size = 5 if scheduler == "slurm" else 1
```

Comment:
why do we need this many for slurm? and why a different number for other scheduler? let's at least drop a comment and if possible let's use fewer nodes.

Reply:
General comment: agreed to keep number of nodes during internal discussion